fixtures/wallet-balance.surql

```diff
@@ -0,0 +1,6 @@
+DEFINE TABLE wallet_balance SCHEMALESS;
```

Comment:
while defining don't we prefer `SCHEMAFULL` instead of `SCHEMALESS`?

Reply:
on point, modified all `.surql` files